projects/storefrontstyles/scss/cxbase/blocks/buttons.scss

```diff
@@ -62,12 +62,11 @@ button {
 
     &.disabled,
     &:disabled {
-      background-color: var(--cx-color-primary);
-      border-color: var(--cx-color-primary);
+      background-color: var(--cx-color-border-focus);
```

Comment:
I think thess styling changes should be documented because of the major release.